### PR TITLE
Stanko tweet model

### PR DESCRIPTION
**Pull request by StankoKuveljic**
- Add Tweet model class
- Make tweet data frame from json row and stream

### CODE REVIEW

File: src/main/scala/io/smartcat/twicas/tweet/Tweet.scala [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package io.smartcat.twicas.tweet
+
+import org.apache.spark.sql.Row
+import twitter4j.Status
+
+/**
```
Comment (mijicd) [inline src/main/scala/io/smartcat/twicas/tweet/Tweet.scala]: Remove autogenerated comments.

File: src/main/scala/io/smartcat/twicas/TrainJob.scala [hunk 2]
```diff
@@ -10,6 +11,15 @@ object TrainJob extends App{
     .appName("twitter_trainer")
     .getOrCreate()
 
-  val ar = spark.sparkContext.parallelize(Array[Int](1, 2, 3, 4, 5, 6, 7))
-  ar.collect().foreach(println)
+  val filename = ""
```
Comment (mijicd) [inline src/main/scala/io/smartcat/twicas/TrainJob.scala:11]: Is this intentionally left empty?

Comment (StankoKuveljic) [reply src/main/scala/io/smartcat/twicas/TrainJob.scala:11]: yes, I'll add this file later.

File: src/main/scala/io/smartcat/twicas/tweet/Tweet.scala (hunk 1 above)
Comment (StankoKuveljic) [reply src/main/scala/io/smartcat/twicas/tweet/Tweet.scala]: fixed

**Review by mijicd — APPROVED** [review]

**Pull request merged**